### DISCUSSION TITLE
Errors when installing SBFspot on Linux Mint 21.2 (Ubuntu 20.04)

### DISCUSSION THREAD

**Discussion by VinceBog** [SBFspot/SBFspot#669] [General]
I'm not able to install SBFspot on a laptop with Linux Mint 21.2 (Ubuntu 20.04) due to multiple errors when I follow the installation manual (error 127, error message 'Not running on Raspberry Pi). Does anyone knows if the installation is possible for a laptop which is no Raspberry Pi?

**Comment by sbf-** [comment]
You have to follow the manual installation method.
See also #665

**Comment by VinceBog** [comment]
`You have to follow the manual installation method.`
I did but that results in error 127...

**Comment by sbf-** [comment]
Without more info I don´t know what E127 is... 
Show some output.

**Comment by VinceBog** [comment]
Error 127 occurs at [Compile SBFspot with SQLite support](https://github.com/SBFspot/SBFspot/wiki/Installation-Linux-SQLite) with the command 'make sqlite'. The response is then:

_test -d sqlite/obj/ || mkdir -p sqlite/obj/
test -d sqlite/bin/ || mkdir -p sqlite/bin/
g++ boost_ext.cpp -c -Wall -O2 -Wno-unused-local-typedefs -Wno-psabi -std=c++11 -DUSE_SQLITE -o sqlite/obj/boost_ext.o 
make: g++: No such file or directory
make: *** [makefile:116: sqlite/obj/boost_ext.o] Error 127_

The maps 'sqlite/obj/' and 'sqlite/bin/' are generated in the map '/home/URSERNAME//SBFspot/SBFspot/' but are empty.

**Reply by sbf-** [reply]
g++ compiler is not installed

`sudo apt install g++`
or
`sudo apt install build-essential`

**Comment by VinceBog** [comment]
Thanks for the reply. 

Error 127 is solved after installing the G++ compiler. Unfortunetly other problems/errors occur when following the manual. I.e. laptop can't make a connection with my Sunny Boy 4000 TL inverter and SBFspotupload status check fails.

I'm afraid this installation is a bit to complicated for a person without Linux real knowledge.
